app/models/contestant_project.rb

```diff
@@ -1,3 +1,16 @@
 class ContestantProject < ApplicationRecord
+  belongs_to :project
+  belongs_to :contestant
+
+  def self.match(int)
+    contestants = []
```

Comment:
Remember to use associations! Anytime you find yourself writing a lot of ruby in a model method, that's usually an indication that you could be using AR and things would be much easier! To grab all the contestants for a given project, you can do `project.contestants` -- our associations allow us to reach through project to grab these objects! `contestants` here will return an array of contestant objects

Reply:
So then we can do `project.contestants.count` to get the count! Using `count` means we should encapsulate this in a model method. So you're on the right track with that, just don't need the ruby before it. This logic could also live in the project model. You could have a method called `contestant_list` which will call `self.contestants.count`